juniper_official/routing/collect-isis-lsdb-tlv.rule

```diff
@@ -0,0 +1,162 @@
+            }
+            field hname {
+                sensor isis-oc {
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/tlvs/tlv/hostname/state/hname";
+                }
+                sensor isis-oc-junos {
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/tlvs/tlv/hostname/state/hostname";
+                }
+                type string;
+                description "hname from the lsdb path";
+            }
+            field router-id {
+                sensor isis-oc {
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/tlvs/tlv/ipv4-te-router-id/state/te-router-id";
+                }
+                sensor isis-oc-junos {
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/tlvs/tlv/ipv4-te-router-id/state/te-router-id";
+                }
+                type string;
+                description "router-id from the lsdb path";
+            }
+            field ipv4-ext-prefix {
+                sensor isis-oc {
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/tlvs/tlv/extended-ipv4-reachability/prefixes/prefix/state/prefix";
+                }
+                sensor isis-oc-junos {
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/tlvs/tlv/extended-ipv4-reachability/prefixes/prefix/state/prefix";
+                }
+                type string;
+                description "ipv4-ext-prefix from the lsdb path";
+            }
+            variable inst-name {
+                value .*;
+                description "Enter a regular expression to match the instance name, e.g., 'master|RI1'";
+                type string;
+            }
+            variable proto-name {
+                value .*;
+                description "Enter a regular expression to match the protocol name, e.g., 'isis' or 'ospf'";
+                type string;
+            }
+            variable level-numb {
+                value .*;
+                description "Enter a regular expression to match the level number, e.g., '1', '2', or '3'";
+                type string;
+            }
+            variable tlv-val {
+                value .*;
+                description "Enter a regular expression to match the TLV value";
+                type string;
+            }
+            rule-properties {
+                version 1;
+                contributor juniper;
+                supported-healthbot-version 4.3.0;
+                supported-devices {
+                    sensors isis-oc;
+                    juniper {
+                        operating-system junos {
+                            sensors isis-oc-junos;						
```

Comment:
All EOP supported platforms and products needs to be added

Reply:
Currently supporting only for MX960 and will be adding other platforms in testing phase.